operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -208,6 +208,38 @@ func TestManifestGenerateGateways(t *testing.T) {
 	checkRoleBindingsReferenceRoles(g, objs)
 }
 
+func TestManifestGenerateIstiodRemote(t *testing.T) {
+	testDataDir = filepath.Join(operatorRootDir, "cmd/mesh/testdata/manifest-generate")
+	g := NewGomegaWithT(t)
+	m, _, err := generateManifest("istiod_remote", "", liveCharts)
+	if err != nil {
+		t.Fatal(err)
+	}
+	objs := parseObjectSetFromManifest(t, m)
+	g.Expect(objs.kind(name.CRDStr).size()).Should(Equal(25))
```

Comment:
hmm, why is this 25?  I don't have any CRD in the istiod-remote

Reply:
my bad, I know now, you had base and istiod remote enabled :)

Reply:
nit: can we just test these exist? Otherwise every time we change the CRDs we need to change this, which has little value